db/migrate/20170502191746_create_invoice_items.rb

```diff
@@ -0,0 +1,14 @@
+class CreateInvoiceItems < ActiveRecord::Migration[5.0]
+  def change
+    create_table :invoice_items do |t|
+      t.string :name
+      t.text :description
+      t.integer :quantity
+      t.integer :unit_price
+      t.datetime :created_at
+      t.datetime :updated_at
+
+      t.timestamps
```

Comment:
Think it would be clearer if we render the timestamps as null: true since we already have created_at and updated_at in our model.